grids/grids.css

```diff
@@ -33,3 +42,11 @@
     grid-template-columns: 1fr;
   }
 }
+
+@media only screen and (max-width: 972px){
```

Comment:
I see you have media queries for `970px` as well as `972px`. They are pretty close values. Could we merge them into one?